tests/snap/semantic_tokens/injected_class_name.cpp

```
@@ -0,0 +1,17 @@
+///
+/// - status: partial
+/// - order: 2
+///
+/// The name itself renders as the class; the constructor reference
+/// currently emits a stray token on the following `(`.
+
+struct Widget {
+    Widget(int size);
+
+    Widget create() {
+        return §Widget§()(42);
```

Cppcheck (2.21.0)

[error] The code contains unhandled character(s) (character code=194). Neither unicode nor extended ascii is supported.

(unhandledChar)